### DISCUSSION TITLE
E800 error path package.json

### DISCUSSION THREAD

**Discussion by GariestGary** [openupm/openupm#3738] [Q&A] [answered]
first time i trying publish package on openupm
https://openupm.com/packages/com.volumebox.toolbox/

Tried many variants to compile build, but still getting this error😭
recreating release, renaming default branch, changing tag dosn't work for me
maybe i missed something important?

**Comment by favoyang** [comment, marked as answer]
Hi @GariestGary,

E800 means our pipeline can not locate a valid package.json with the given package name. It happens either with a package rename, or a broken package.json (syntax error).

It looks like you should quote these version numbers as strings.

```
2.16.0 => "2.16.0"
```

![image](https://user-images.githubusercontent.com/125390/219588012-b6554a0c-fb03-49f9-aa34-9577fb6954ed.png)

**Reply by GariestGary** [reply]
🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♂️🤦‍♂️🤦‍♂️ THANKS! It now works as it should!
